src/packages/core/tests/connector.test.ts

```diff
@@ -4,12 +4,16 @@ import { Provider as EthereumProvider } from "@ganache/ethereum";
 
 describe("connector", () => {
   it("works without passing options", async () => {
-    assert.doesNotThrow(() => Ganache.provider());
+    assert.doesNotThrow(async () => {
+      const provider = Ganache.provider();
+      await provider.once("connect");
```

Comment:
Is it required to `await provider.once("connect");` for these tests now?

Reply:
No it's not required; I can remove them if you want. I figured it'd be good practice to use it, but if you'd prefer to use the request coordinators queuing logic in all the tests ok. I personally believe that should be a fallback rather than the main use case and we should be telling users to explicitly wait for Ganache to initialize in default use cases.

Regardless, I can also add a pair of tests that ensure the async logic here works properly.

Reply:
I'd prefer to reduce boilerplate, rather than add to it. I'd prefer they awaits were removed.

And you know I always appreciate more tests! :-D